debian/control

```diff
@@ -0,0 +1,8 @@
+Package: capstone
+Version: 3.1.0
```

Comment:
~~This should be v6.0.0-alpha I think. Not sure how Debian likes it's versioning. But this is the latest released version.~~
Sorry, just saw you already do this. Nice! Please replace the version number with something like `<version-placeholder>` or similar. So we are not confused.